DXMainClient/DXGUI/Multiplayer/CnCNet/CnCNetGameLoadingLobby.cs

```diff
@@ -640,5 +645,21 @@ public override string GetSwitchName()
         {
             return "Load Game";
         }
+
+        protected override void UpdateDiscordPresence(bool resetTimer = false)
+        {
+            if (discordHandler == null)
+                return;
+
+            PlayerInfo player = Players.Find(p => p.Name == ProgramConstants.PLAYERNAME);
+            if (player == null || lblMapNameValue == null || lblGameModeValue == null)
```

Comment:
`lblMapNameValue` and `lblGameModeValue` are UI controls, they can never be null.

Reply:
Ditto.